docs/source/operations/projections/peirce_q.rst

```diff
@@ -21,12 +39,33 @@ Peirce Quincuncial
 +---------------------+----------------------------------------------------------+
 
 
-.. figure:: ./images/peirce_q.png
+.. figure:: ./images/peirce_q_square.png
+   :width: 500 px
+   :align: center
+   :alt:   Peirce Quincuncial (Square)
+
+   proj-string: ``+proj=peirce_q +lon_0=25 +type=square``
+
+.. figure:: ./images/peirce_q_diamond.png
    :width: 500 px
    :align: center
-   :alt:   Peirce Quincuncial
+   :alt:   Peirce Quincuncial (Diamond)
+
+   proj-string: ``+proj=peirce_q +lon_0=25 +type=diamond``
+
+.. figure:: ./images/peirce_q_horizontal.png
+   :width: 500 px
+   :align: center
+   :alt:   Peirce Quincuncial (Horizontal)
+
+   proj-string: ``+proj=peirce_q +lon_0=25 +type=horizontal``
+
+   .. figure:: ./images/grieger_triptychial.png
+      :width: 500 px
+      :align: center
+      :alt:   Grieger Triptychial
 
-   proj-string: ``+proj=peirce_q``
+      proj-string: ``+proj=pipeline +step +proj=ob_tran +o_proj=peirce_q +o_lat_p=-45 +o_lon_p=45 +type=horizontal +o_scrollx=-0.25 +step +proj=affine +s11=-1 +s12=0 +s21=0 +s22=-1``
```

Comment:
+type should be +o_type, otherwise it won't be forwarded by ob_tran
And the +o_scrollx will be translated by ob_tran as +scrollx

Reply:
That's not how it's working for me: type is being passed directly to peirce_q. If I try o_type it fails to pass on the param. (Same now that I have changed o_scrollx to scrollx).

Reply:
> That's not how it's working for me: type is being passed directly to peirce_q. If I try o_type it fails to pass on the param. (Same now that I have changed o_scrollx to scrollx).

oh, I'm sorry: I was confused. So checking at how ob_tran actually works, it will forward "type", "scrollx", "scrolly" unmodified. Ignore contradicting statements I can have put regarding this.

Reply:
ok! no problem!